src/lib/DataTable.tsx

```diff
@@ -321,7 +321,7 @@ class TableHead<T> extends React.Component<TableHeadProps<T>> {
         };
     };
     _getRenderedColumn = (index?: number) => {
-        if (index) {
```

Comment:
Previously this condition was false for the first column, so resize for this column didn't work